resources/views/welcome.blade.php

```diff
@@ -1,7 +1,7 @@
 @extends('layouts.app')
```

Comment:
there are a lot of elements with `.ris-welcome__people` which are basically the same, could be moved in a separate file/component